index.html

```diff
@@ -2,316 +2,349 @@
+                            <img src="images/githublogo.png" alt="Github icon" />
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="images/Vector (2).svg" alt="two fingers"></a>
+                    </li>
+                </ul>
+                <button type="button" class="resume-button button">Get my resume</button>
+            </section>
+        </section>
+        <div class="languages_for_web">
+            <section class="languages">
+                <a href="#" class="languages-heading">
+                    <h3 class="font-style">Languages</h3>
+                    <img class="vector" src="images/arrow-down.png" alt="arrow down" />
+                </a>
+                <ul class="lang">
+                    <li>
+                        <a class="language" href="#lang">
+                            <img src="images/js.svg" alt="javascript logo" />
+                            <h4 class="language-text javascript font-style">
+                                JavaScript
+                            </h4>
+                        </a>
+                    </li>
+                    <li>
+                        <a class="language m-left" href="#lang">
+                            <img src="images/html.svg" alt="html logo" />
+                            <h4 class="language-text html font-style">HTML</h4>
+                        </a>
+                    </li>
+                    <li>
+                        <a class="language m-left" href="#lang">
+                            <img src="images/css.svg" alt="css logo" />
+                            <h4 class="language-text html font-style">CSS</h4>
+                        </a>
+                    </li>
+                </ul>
+            </section>
+            <section class="frameworks">
+                <div class="framework-list">
+                    <h3 class="font-style">Frameworks</h3>
+                    <img class="vector" src="images/right-arrow.png" alt="arrow right" />
+                </div>
+                <div class="framework-list">
+                    <h3 class="font-style">Skills</h3>
+                    <img class="vector" src="images/right-arrow.png" alt="arrow right" />
+                </div>
+            </section>
+        </div>
+
+
+    </div>
+
+    <footer id="contact" class="footer">
+        <div class="contact-wrapper">
+            <h2 class="contact font-style">Contact me</h2>
+            <p class="contact-description font-style">
+                If you have an application you are interested in developing, a
+                feature that you need built or a project that needs coding. I’d love
+                to help with it
+            </p>
+            <form id="contact-form" action="https://formspree.io/f/xrgjnypg" method="post" class="form-wrapper">
+                <fieldset class="form-fieldset">

```

Comment:
- [ ] Kindly seperate the footer tag from the form tag ,becouse the form is seperate section with the contact me section kindly fix this for proper semantic reason.